docs/next.config.js

```diff
@@ -5,13 +5,22 @@ const semver = require('semver');
 
 const { version } = require('./package.json');
 
+const { schema_sync } = require('./scripts/schema-sync');
+
 // copy versions/v(latest version) to versions/latest
 // (Next.js only half-handles symlinks)
 const vLatest = join('pages', 'versions', `v${version}/`);
 const latest = join('pages', 'versions', 'latest/');
 removeSync(latest);
 copySync(vLatest, latest);
 
+// Starting with version 38, download xdl schema at build time
+// (and when starting a dev server)
+for (let i = 38; i <= parseInt(version, 10); i++) {
```

Comment:
I think it might be "more stable" or better if you use the versions from within the docs. Instead of hardcoding 38, you can also check on the `const { VERSIONS } = require('./common/versions.js');` I think. With these, you know exactly which versions are still supported and only download these 😄